status-server/src/index.js

```diff
@@ -32,38 +32,58 @@ app.get('/user/:id/status', function (req, res) {
       return res.status(500).send('Error reading status!');
     }
 
-    fs.stat(filename, function (err, stats) {
-      if (err) {
-        console.error(err.stack);
-        return res.status(500).send('Error reading status last-modified!');
-      }
-
-      res.setHeader('Last-Modified', stats.mtime);
-      res.json({ 'last-modified': stats.mtime, ok: true, value: buf.toString() });
-    });
+      res.json(JSON.parse(buf.toString()));
+  });
+});
+
+app.get('/user/:id/message', function (req, res) {
+  var num = req.params.id || 0;
+  var filename = path.join(dataDir, 'user', num, 'message');
+
+  fs.readFile(filename, function (err, buf) {
```

Comment:
This is only going to work once all the data files are updated to contain the JSON data. Can you update each of them and add them to this pull request?